build/web-client-rpc.md

```diff
@@ -0,0 +1,55 @@
+# Web Client X RPC
```

Comment:
Other suggestions:
- "Web Client VS RPC Client"
- "Comparing Web Client and RPC Client" to interact with the Nimiq Blockchain

> I am not a big fan of the "X"